homeassistant/components/media_player/webostv.py

```diff
@@ -87,6 +87,7 @@ def setup_tv(host, mac, name, customize, config, hass, add_devices):
     from pylgtv import WebOsClient
     from pylgtv import PyLGTVPairException
     from websockets.exceptions import ConnectionClosed
+    from concurrent import futures
```

Comment:
ok  thanks, I had a bit a problem figuring out what the right way to do this was